.github/workflows/update-docs-in-website.yml

```diff
@@ -0,0 +1,50 @@
+name: Update latest Extensions documentation in the website
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'docs/*.md'
+
+jobs:
+  Make-PR:
+    name: Make PR on website repository with updated latest Extensions documentation
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - name: Checkout Current repository
+        uses: actions/checkout@v3
+        with:
+          path: extensions
+      - name: Checkout Another repository
+        uses: actions/checkout@v3
+        with:
+          repository: asyncapi/website
+          path: website
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Config git
+        run: |
+          git config --global user.name asyncapi-bot
+          git config --global user.email info@asyncapi.io
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-extensions-docs-${{ github.sha }}
+      - name: Copy extensions folder from Current Repo to Another
+        working-directory: ./website
+        run: |
+          mkdir -p ./pages/docs/reference/extensions
+          printf "%s\ntitle: Extensions\nweight: 10\n%s" "---" "---"> ../extensions-catalog/extensions/_section.md
```

Comment:
@derberg need your help here 
as here the actions doesn't move in the 'extensions' folder